invisible_cities/cities/buffy_test.py

```diff
@@ -72,31 +72,48 @@ def test_buffy_kr(config_tmpdir, full_sim_file):
         assert np.all(sipm_integ[sipm_nonzero] == sns_sums[sns_sums.index > 12])
 
 
-def test_buffy_filters_empty(config_tmpdir, ICDATADIR):
+def test_buffy_no_file_without_sns_response(config_tmpdir, ICDATADIR):
     file_in  = os.path.join(ICDATADIR, 'nexus_new_kr83m_fast.oldformat.sim.h5')
-    file_out = os.path.join(config_tmpdir, 'filtered_empty.buffers.h5')
+    file_out = os.path.join(config_tmpdir, 'no_file.h5')
 
     nevt = 2
     conf = configure('buffy invisible_cities/config/buffy.conf'.split())
     conf.update(dict(files_in=file_in, file_out=file_out, event_range=nevt))
 
-    # Warning expected since no MC tables present.
+    try:
+        buffy(**conf)
+    except:
+        if not os.path.exists(file_out):
+            assert True
+        else:
+            assert False
+
+
+def test_buffy_filters_empty(config_tmpdir, ICDATADIR):
+    file_in  = os.path.join(ICDATADIR, 'nexus_new_kr83m_*.oldformat.sim.h5')
```

Comment:
when #877 is merged you should be able to specify the input files without a wildcard, which will make this more clear.

Regarding the treatment of files with missing sensor response, I suggest we add a parameterized test with both file orders (the offending file first or last) and mark the order that doesn't work as `xfail` until we fix it.

Remember to add a docstring documenting what the test does and to comment that we use the fast file to mimic a full file without sensor responses.

Remove the commented line below.

The variable should be renames to `files_in` (with an s)